practice-app/back-end/routes/keys.js

```diff
@@ -3,4 +3,8 @@ module.exports = {
 	consumer_secret: '',
 	access_token_key: '',
 	access_token_secret: ''
-}
\ No newline at end of file
+<<<<<<< HEAD
+}
+=======
+}
+>>>>>>> origin
```

Comment:
I think these lines should be replaced by a single closing }

Reply:
These lines were forgetted while merging. It is updated.

Reply:
These lines were forgetted while merging. It is updated.

Reply:
These lines were forgotten while merging. It is updated.